recipes/cis/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.3.4" %}
+
+package:
+  name: cis
+  version: {{ version }}
+
+source:
+  fn: cis-{{ version }}.tar.gz
+  url: https://github.com/cedadev/cis/archive/{{ version }}.tar.gz
+  sha256: 553d92ae114da80090125b1b057813e0cd9a4366000d233eb875f43e92fd0ac0
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - matplotlib >=1.2.0
```

Comment:
You will have to add `setuptools` here to use https://github.com/conda-forge/staged-recipes/pull/556/files#r62188627